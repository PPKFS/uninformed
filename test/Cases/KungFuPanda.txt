[From the pen of Mike Gentry. The interesting point is "add Panda at entry
6 in the Furious Five, if absent.]

The Jade Temple Dojo is a room. "Here is where the great kung fu masters --
[Furious Five] -- practice their ancient fighting arts."

Tiger is an animal, here. Snake is an animal, here. Mantis is an animal,
here. Crane is an animal, here. Monkey is an animal, here.
 
The Furious Five is a list of animals that varies.
 
When play begins:
	repeat with kung fu master running through animals in the Jade Temple Dojo:
		add the kung fu master to the Furious Five.
 
Panda is an animal.
 
The dragon scroll is carried by the player.
 
Instead of examining the dragon scroll:
	say "What is this? According to the scroll, a humble Panda is destined to
		defeat the mighty Tai Lung!";
	move Panda to the Jade Temple Dojo;
	add Panda at entry 6 in the Furious Five, if absent.
 
Test me with "x scroll / l / x scroll / l".
