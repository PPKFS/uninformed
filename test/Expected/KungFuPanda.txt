149 words
¶|__¶ ¶the¶ ¶jade¶ ¶temple¶ ¶dojo¶ ¶is¶ ¶a¶ ¶room¶ ¶.¶ ¶"Here is where the great kung fu masters --
[Furious Five] -- practice their ancient fighting arts."¶ ¶tiger¶ ¶an¶ ¶animal¶ ¶,¶ ¶here¶ ¶snake¶ ¶mantis¶ ¶crane¶ ¶monkey¶ ¶furious¶ ¶five¶ ¶list¶ ¶of¶ ¶animals¶ ¶that¶ ¶varies¶ ¶when¶ ¶play¶ ¶begins¶ ¶:¶ ¶repeat¶ ¶with¶ ¶kung¶ ¶fu¶ ¶master¶ ¶running¶ ¶through¶ ¶in¶ ¶add¶ ¶to¶ ¶panda¶ ¶dragon¶ ¶scroll¶ ¶carried¶ ¶by¶ ¶player¶ ¶instead¶ ¶examining¶ ¶say¶ ¶"What is this? According to the scroll, a humble Panda is destined to
defeat the mighty Tai Lung!"¶ ¶;¶ ¶move¶ ¶at¶ ¶entry¶ ¶6¶ ¶if¶ ¶absent¶ ¶test¶ ¶me¶ ¶"x scroll / l / x scroll / l"¶ 
60 distinct words
¶|__¶ ¶|__¶ 0a
¶The¶ ¶the¶ 20
¶Jade¶ ¶jade¶ 20
¶Temple¶ ¶temple¶ 20
¶Dojo¶ ¶dojo¶ 20
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶room¶ ¶room¶ 20
¶.¶ ¶.¶ 20
¶"Here is where the great kung fu masters --
[Furious Five] -- practice their ancient fighting arts."¶ ¶"Here is where the great kung fu masters --
[Furious Five] -- practice their ancient fighting arts."¶ 20
¶|__¶ ¶|__¶ 0a
¶Tiger¶ ¶tiger¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶,¶ ¶,¶ 20
¶here¶ ¶here¶ 20
¶.¶ ¶.¶ 20
¶Snake¶ ¶snake¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶,¶ ¶,¶ 20
¶here¶ ¶here¶ 20
¶.¶ ¶.¶ 20
¶Mantis¶ ¶mantis¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶,¶ ¶,¶ 20
¶here¶ ¶here¶ 0a
¶.¶ ¶.¶ 20
¶Crane¶ ¶crane¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶,¶ ¶,¶ 20
¶here¶ ¶here¶ 20
¶.¶ ¶.¶ 20
¶Monkey¶ ¶monkey¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶,¶ ¶,¶ 20
¶here¶ ¶here¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶The¶ ¶the¶ 20
¶Furious¶ ¶furious¶ 20
¶Five¶ ¶five¶ 20
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶list¶ ¶list¶ 20
¶of¶ ¶of¶ 20
¶animals¶ ¶animals¶ 20
¶that¶ ¶that¶ 20
¶varies¶ ¶varies¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶When¶ ¶when¶ 20
¶play¶ ¶play¶ 20
¶begins¶ ¶begins¶ 20
¶:¶ ¶:¶ 20
¶repeat¶ ¶repeat¶ 41
¶with¶ ¶with¶ 20
¶kung¶ ¶kung¶ 20
¶fu¶ ¶fu¶ 20
¶master¶ ¶master¶ 20
¶running¶ ¶running¶ 20
¶through¶ ¶through¶ 20
¶animals¶ ¶animals¶ 20
¶in¶ ¶in¶ 20
¶the¶ ¶the¶ 20
¶Jade¶ ¶jade¶ 20
¶Temple¶ ¶temple¶ 20
¶Dojo¶ ¶dojo¶ 20
¶:¶ ¶:¶ 20
¶add¶ ¶add¶ 42
¶the¶ ¶the¶ 20
¶kung¶ ¶kung¶ 20
¶fu¶ ¶fu¶ 20
¶master¶ ¶master¶ 20
¶to¶ ¶to¶ 20
¶the¶ ¶the¶ 20
¶Furious¶ ¶furious¶ 20
¶Five¶ ¶five¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Panda¶ ¶panda¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶animal¶ ¶animal¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶The¶ ¶the¶ 20
¶dragon¶ ¶dragon¶ 20
¶scroll¶ ¶scroll¶ 20
¶is¶ ¶is¶ 20
¶carried¶ ¶carried¶ 20
¶by¶ ¶by¶ 20
¶the¶ ¶the¶ 20
¶player¶ ¶player¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Instead¶ ¶instead¶ 20
¶of¶ ¶of¶ 20
¶examining¶ ¶examining¶ 20
¶the¶ ¶the¶ 20
¶dragon¶ ¶dragon¶ 20
¶scroll¶ ¶scroll¶ 20
¶:¶ ¶:¶ 20
¶say¶ ¶say¶ 41
¶"What is this? According to the scroll, a humble Panda is destined to
defeat the mighty Tai Lung!"¶ ¶"What is this? According to the scroll, a humble Panda is destined to
defeat the mighty Tai Lung!"¶ 20
¶;¶ ¶;¶ 20
¶move¶ ¶move¶ 41
¶Panda¶ ¶panda¶ 20
¶to¶ ¶to¶ 20
¶the¶ ¶the¶ 20
¶Jade¶ ¶jade¶ 20
¶Temple¶ ¶temple¶ 20
¶Dojo¶ ¶dojo¶ 20
¶;¶ ¶;¶ 20
¶add¶ ¶add¶ 41
¶Panda¶ ¶panda¶ 20
¶at¶ ¶at¶ 20
¶entry¶ ¶entry¶ 20
¶6¶ ¶6¶ 20
¶in¶ ¶in¶ 20
¶the¶ ¶the¶ 20
¶Furious¶ ¶furious¶ 20
¶Five¶ ¶five¶ 20
¶,¶ ¶,¶ 20
¶if¶ ¶if¶ 20
¶absent¶ ¶absent¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Test¶ ¶test¶ 20
¶me¶ ¶me¶ 20
¶with¶ ¶with¶ 20
¶"x scroll / l / x scroll / l"¶ ¶"x scroll / l / x scroll / l"¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
